core/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala

```diff
@@ -104,24 +108,30 @@ abstract class LogStoreSuiteBase extends QueryTest
 
   test("listFrom") {
     val tempDir = Utils.createTempDir()
+    val tempPath = new Path(tempDir.toURI)
     val store = createLogStore(spark)
 
-    val deltas =
-      Seq(0, 1, 2, 3, 4).map(i => new File(tempDir, i.toString)).map(_.toURI).map(new Path(_))
-    store.write(deltas(1), Iterator("zero"), overwrite = false, sessionHadoopConf)
-    store.write(deltas(2), Iterator("one"), overwrite = false, sessionHadoopConf)
-    store.write(deltas(3), Iterator("two"), overwrite = false, sessionHadoopConf)
+    def toDelta(n: Int): Path = FileNames.deltaFile(tempPath, n)
+
+    val deltas = Seq(0, 1, 2, 3, 4).map(toDelta)
+
+    // let's call write with overwrite = true to skip additional checks
+    // done by some implementations (like DynamoDBLogStore)
```

Comment:
why do we need this change? Since there is no `DynamoDBLogStoreSuite`, and instead we use the `ExternalLogStoreSuite` which uses the `InMemoryLogStore` ... are these additional checks an issue?